graylog2-server/src/main/java/org/graylog/security/authservice/backend/ADAuthServiceBackend.java

```diff
@@ -98,12 +98,17 @@ public ADAuthServiceBackend(UnboundLDAPConnector ldapConnector,
                     return Optional.empty();
                 }
             }
+            if (!userEntry.accountIsEnabled()) {
```

Comment:
How about moving this up in front of the `if (!authCredentials.isAuthenticated()) {` line? That way we get the warning when a disabled user tries to log in. Otherwise we will never see the warning because the authentication doesn't work.

Reply:
sure. makes sense